src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/CreateClaimCallbackHandler.java

```diff
@@ -432,6 +468,18 @@ private String getBody(CaseData caseData) {
     }
 
     private List<String> validateCaseData(CaseData caseData) {
+        List<String> errorsMessages = new ArrayList<>();
+        // Tactical fix. We have an issue where null courtLocation is being submitted.
+        // We are validating it exists on submission if not we return an error to the user.
+        if (caseData.getCourtLocation() == null
+            || caseData.getCourtLocation().getApplicantPreferredCourtLocationList() == null
+            || caseData.getCourtLocation().getApplicantPreferredCourtLocationList().getValue() == null) {
+            errorsMessages.add("Court location code is required");
+        }
+        return errorsMessages;
+    }
+
+    private List<String> validateCaseDataOld(CaseData caseData) {
```

Comment:
Not important, but if these methods were, for instance validateCourtText and validateCourtChoice I think they'd be understood without the need to look at what they do

Reply:
hey, these are supposed to be deleted thats why the 'old', it's only there for backwards compatibility